.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+combine_as_imports=True
+line_length=88
```

Comment:
Add a newline here. My OCD behind this. Sorry!

Reply:
No new line there xD

Reply:
And squash em

Reply:
No squashing either. Config files must have their commits. Incase something wrong happens :'(

Reply:
Ohh yes! Black-isort fights fixed I guess

Reply:
No guessing. It did B)